keps/sig-api-machinery/1965-kube-apiserver-identity/README.md

```diff
@@ -179,7 +178,7 @@ Alpha should provide basic functionality covered with tests described above.
 
 #### Beta -> GA Graduation
 
-==TODO==
+  - support for aggregated apiservers (with sufficient integration/e2e test coverage)
```

Comment:
We'll need to document how to implement this in your aggregated API server. The docs should cover what's needed even if your implementation isn't in Go. It doesn't need to be in any great detail but I'm convinced we should make it feasible for an author to at least learn what they must implement themselves in order to conform.